routes/api/v1/forecast.js

```diff
@@ -1,72 +1,14 @@
 var express = require('express');
 var router = express.Router();
-var User = require('../../../models').User;
-require('dotenv').config(); // Loads environment variables from .env file
-const https = require('https');
```

Comment:
@plapicola why the choice to pull this out? Did you end up moving it somewhere else?

Reply:
I wound up moving the user down to the facade, the dotenv files to the service, and in the case of https, ditched it entirely in favor of node-fetch.